lib/crudry_query.ex

```diff
@@ -29,17 +29,14 @@ defmodule Crudry.Query do
     offset = Keyword.get(opts, :offset, 0)
     sorting_order = Keyword.get(opts, :sorting_order, :asc)
     order_by = Keyword.get(opts, :order_by)
-    order = if order_by, do: [{sorting_order, to_atom(order_by)}], else: []
+    order = if order_by, do: parse_order_by_args(sorting_order, order_by), else: []
```

Comment:
Since we now have a function to parse order by, I think we can remove the `if` here and deal with it using pattern matching